docker-compose.yml

```diff
@@ -25,22 +28,22 @@ services:
         constraints: [node.role == manager]
   ui:
     image: dockervoyagerlabs/dfproxy:1.7.4
-    ports:
-      - 3333:3333
     networks:
-      - default
+      - proxy
     deploy:
       placement:
         constraints: [node.role == manager]
       labels:
         - com.df.distribute=true
         - com.df.notify=true
-        - com.df.servicePath=/
+        - com.df.servicePath=/ui
```

Comment:
This does not work (and it should). A better alternative would be to define an environment variable like `ROOT_PATH=/ui` (or any other value).